inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -634,9 +635,10 @@ public void onRecommendationRejectEvent(AjaxRecommendationRejectedEvent aEvent)
         
         if (sessionActive && eventState.getUser().equals(annotatorState.getUser())
                 && eventState.getProject().equals(annotatorState.getProject())) {
-            if (eventState.getDocument().equals(annotatorState.getDocument())
+            SourceDocument document = eventState.getDocument();
+            if (document.equals(annotatorState.getDocument())
                     && aEvent.getVid().getLayerId() == selectedLayer.getObject().getId()
-                    && predictionModel.getPredictionByVID(aEvent.getVid())
```

Comment:
The `getPredictionByVID` method seems to be fundamentally flawed, so IMHO it should be removed and replaced by the `getPredictionInDocumentByVID` method. It seems you only add the `getPredictionInDocumentByVID` and the old one still sticks around and could cause further trouble.